core/command.go

```diff
@@ -94,3 +96,9 @@ func (GoBack) Execute(oldState State) (State, error) {
 		history:   newHistory,
 	}, nil
 }
+
+func (m Mark) Execute(oldState State) (State, error) {
+	newState := copyState(oldState)
+	newState.Folder.Files[newState.Selected].ToggleIsMarked()
```

Comment:
Valid point. I will change that and also add new code which uses different color for a marked file's line. Update coming tomorrow.